wa/framework/output.py

```diff
@@ -81,6 +81,12 @@ def events(self):
             return []
         return self.result.events
 
+    @property
+    def metadata(self):
```

Comment:
This should probably be added to the API documentation as well.

Reply:
It's already documented in the API as being there:

https://github.com/ARM-software/workload-automation/blob/next/doc/source/api/output.rst#id5

just was never actually exposed in the output -- whoops!